psl/psl-core/src/datamodel_connector.rs

```diff
@@ -73,6 +73,9 @@ pub trait Connector: Send + Sync {
     /// The referential actions supported by the connector.
     fn referential_actions(&self, relation_mode: &RelationMode) -> BitFlags<ReferentialAction>;
```

Comment:
Technically, `psl_core::datamodel_connector::Connector::referential_actions` no longer needs the `relation_mode` argument.